kevinzs/day01/tests.py

```diff
@@ -0,0 +1,29 @@
+import day01
+import unittest
+
+class Day01Tests(unittest.TestCase):
+
+	def test_part_one(self):
```

Comment:
You can see the part of the tests in @JoseLlorensRipolles Review with a tips for this kind of tests 👍